packages/event-overview/src/pages/events.tsx

```diff
@@ -120,9 +121,13 @@ export function EventsPage({}: RouteComponentProps) {
           {loading ? <Spinner /> : null}
 
           {viewType === 'tile' ? (
-            <CardWrapper>
-              <EventTiles events={events} />
-            </CardWrapper>
+            events && events.length > 0 ? (
+              <CardWrapper>
+                <EventTiles events={events} />
+              </CardWrapper>
+            ) : (
+              <EmptyState />
```

Comment:
Maybe add this to the `EventTiles` component? When events is empty, render empty state